### DISCUSSION TITLE
Toggle or delay the /ClaimExplosions reminder in chat

### DISCUSSION THREAD

**Discussion by Tosfoy** [GriefPrevention/GriefPrevention#2211] [Ideas]
Hello!
My players are huge on making map arts and they are getting spammed in chat for each tnt placed while in their claim
Enabling it is not the way to go for them and we'd love a feature to either delay the reminder, or just turn it off completely
Something in the config like
RemindClaimExplosive: False
ClaimExplosiveMessageDelay: 300 (seconds)
![image](https://github.com/GriefPrevention/GriefPrevention/assets/58991099/352e15f9-efe1-4eac-bed3-fbae55c96da7)

Thank you!

**Comment by RoboMWM** [comment]
Cooldown implemented in https://github.com/GriefPrevention/GriefPrevention/discussions/2211

**Reply by RoboMWM** [reply]
Do note he did not PR to legacy, so it won't be included in any official releases until v20 hits. If you want it in v16 releases, you should probably ping him to do so. It's a small, maintainable change so I'd very likely approve it.
